src/PubSub/Subscription.php

```diff
@@ -223,6 +223,52 @@ public function create(array $options = [])
     }
 
     /**
+     * Update the subscription.
+     *
+     * Note that subscription name and topic are immutable properties and may
```

Comment:
Should we add a whitelist notification?

Reply:
Yes - I thought that could be part of the followup PR to support updateTopic and updateSnapshot, because it may also require additional changes such as whitelist tests. If you prefer, I can add here.

Reply:
Adding it in the follow up PR sounds good 😄